src/System.Runtime/tests/System/Uri.CreateStringTests.cs

```diff
@@ -549,8 +549,16 @@ public static IEnumerable<object[]> Path_Query_Fragment_TestData()
             // File with host
             yield return new object[] { "file://path1/path2", "/path2", "", "" };
             yield return new object[] { "file:///path1/path2", "/path1/path2", "", "" };
-            yield return new object[] { @"file:///path1\path2/path3\path4", "/path1/path2/path3/path4", "", "" };
-            yield return new object[] { @"file:///path1\path2/path3\path4\", "/path1/path2/path3/path4/", "", "" };
+            if (s_isWindowsSystem)
```

Comment:
This test was already there, but now its result changed due to the backwards slash being maintained.
Should I add the same test to another project? which one?